pyuntl/dc_structure.py

```diff
@@ -1,15 +1,19 @@
 import string
+
 from pyuntl import DC_ORDER
 
+
 XSI = 'http://www.w3.org/2001/XMLSchema-instance'
 
-# Namespaces for the DC xml
+
+# Namespaces for the DC XML.
 DC_NAMESPACES = {
     'oai_dc': 'http://www.openarchives.org/OAI/2.0/oai_dc/',
     'dc': 'http://purl.org/dc/elements/1.1/',
     'xsi': XSI,
 }
 
+
```

Comment:
Here you use two blank lines between constants, but in etd_ms_structure.py, there is only a single blank line separating them. We should make the spacing consistent across all these files.